src/lib.rs

```diff
@@ -157,7 +157,7 @@ mod filter;
 mod filter;
 
 /// Log target, either stdout or stderr.
-#[derive(Debug)]
+#[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
```

Comment:
Hmm, `Ord` and `PartialOrd` probably don't make sense for this `enum`. I think we should remove them, but the rest look good.

Reply:
Sorry, I'd missed your comment on #6. I agree: I don't think they make sense, so I've removed them in the latest commit.